### ISSUE TITLE
Request to add HALC in multimodal hallucination

### ISSUE THREAD

**Issue by BillChan226** [BradyFU/Awesome-Multimodal-Large-Language-Models#130]
Hi Brady,

First, thanks for the valuable effort in summarizing these incredible works together! 

Would you please also consider including our recent work HALC in your repo?
Paper title: [HALC: Object Hallucination Reduction via Adaptive Focal-Contrast Decoding](https://arxiv.org/abs/2403.00425)
Paper link: https://arxiv.org/pdf/2403.00425.pdf
Repo link: https://github.com/BillChan226/HALC
Project page: https://billchan226.github.io/HALC
The short version of this paper has been accepted by ICLR R2-FM workshop, and the long version is still under review.

Thank you for your kind consideration!

**Comment by xjtupanda** [comment]
Thanks for sharing! Your work has been added.

**Comment by BillChan226** [comment]
> Thanks for sharing! Your work has been added.

Thank you very much!
